coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetViewHelperServiceImpl.java

```diff
@@ -205,4 +207,15 @@ public void setKcBusinessRulesEngine(KcBusinessRulesEngine kcBusinessRulesEngine
 		this.kcBusinessRulesEngine = kcBusinessRulesEngine;
 	}
     
+	public boolean isBudgetMarkedForSubmission() {
+		ProposalBudgetForm form = (ProposalBudgetForm)ViewLifecycle.getModel();
+		boolean budgetMarkedForSubmission = false;
+		Budget finalBudget = form.getDevelopmentProposal().getFinalBudget();
```

Comment:
At least pass in the Budget or DevelopmentProposal as an argument so we aren't relying on global memory which makes it easier to make out to a service at a later for other uses.